apps/fluent-tester/src/FluentTester/TestComponents/Button/ButtonIconTest.tsx

```diff
@@ -20,20 +20,27 @@ export const ButtonIconTest: React.FunctionComponent<{}> = () => {
     viewBox: '0 0 500 500',
   };
 
+  // SVG-based icons are not available on all platforms yet
+  const svgIconsEnabled = ['ios', 'macos', 'win32'].includes(Platform.OS as string);
```

Comment:
Maybe a place for @tamasane or @mnkshukl to follow up if / when Android works

Reply:
Also, I think SVG's work with react-native-web as well? Probably not blocking to include that since nothing really depends on the web platform...

Reply:
Yep, and once new platforms get verified we can add them here.